assets/sass/components/key-metrics/_googlesitekit-km-selection-panel.scss

```diff
@@ -112,6 +120,10 @@
 		column-gap: $grid-gap-desktop;
 		display: flex;
 
+		@media (max-width: $bp-xsmallOnly) {
+			justify-content: center;
+		}
+
```

Comment:
While this makes sense, only applying it up to `$bp-xsmallOnly` causes a broken layout for viewports between 450px to 599px. See:

![image](https://github.com/google/site-kit-wp/assets/20284937/4fd7fcc8-9297-420c-a33b-f9a5e0bcd154)

I think it is fine if we just apply this style for all viewports.